core/declarations.gms

```diff
@@ -605,9 +597,12 @@ s_MtCO2_2_GtC                                         "conversion factor from Mt
 
 s_MtCH4_2_TWa                                        "Energy content of methane. MtCH4 --> TWa: 1 MtCH4 = 1.23 * 10^6 toe * 42 GJ/toe * 10^-9 EJ/GJ * 1 TWa/31.536 EJ = 0.001638 TWa (BP statistical review)"  /0.001638/
 
-sm_h2kg_2_h2kWh                                      "convert kilogramme of hydrogen to kwh energy value." /32.5/
+s_D2015_2_D2017                                         "Convert $2015 to $2017"      /1.0292/
```

Comment:
Great, a new type style, as USD, US$, US, USD_ and EUR, EUR_ used in our output files were not enough 🥳